docs/src/Microphysics2M.md

```diff
@@ -1,12 +1,255 @@
+\end{equation}
+```
+where:
+  - ``q_{liq}`` is the cloud liquid water specific humidity,
+  - ``\rho`` is the moist air density,
+  - ``k_c`` is the Long's collection kernel constant,
+  - ``\nu`` is the cloud droplet gamma distribution parameter,
+  - ``\left. \frac{d \, N_{liq}}{dt} \right|_{acnv}`` is the rate of change of cloud droplets number density by autoconversion.
+
+### Raindrops self-collection
+
+An approximation for rate of change of raindrops number density due to self-collection is obtained by directly evaluating the integral:
+```math
+\begin{align}
+    \left. \frac{d \, N_{rai}}{dt} \right|_{sc}= -\frac{1}{2}\int_{x=0}^{\infty}\int_{y=0}^\infty f_r(x) f_r(y) K(x,y) dy dx.
+\end{align}
+```
+This yields,
+```math
+\begin{equation}
+  \left. \frac{d \, N_{rai}}{dt} \right|_{sc} = -k_r N_{rai} (q_{rai} \rho),
+\end{equation}
+```
+where:
+  - ``q_{rai}`` is the rain water specific humidity,
+  - ``\rho`` is the moist air density,
+  - ``N_{rai}`` is the raindrops number density,
+  - ``k_r`` is the Long's collection kernel constant.
+
+## Terminal velocity and evaporation
+
+The double-moment parametrization of terminal velocity and rain evaporation is based on the single-moment parametrization. We assume the traditional exponential size distribution for raindrops:
+```math
+\begin{equation}
+n(r) = n_0 exp(-\lambda r),
+\end{equation}
+```
+where ``n_0`` and ``\lambda`` are the distribution parameters. Unlike the single-moment parametrization where ``n_0`` is assumed constant, here, we derive an equation for ``n_0``. By integrating the exponential size distribution for the first moment we find:
+```math
+\begin{equation}
+n0 = N_{rai} * \lambda,
+\end{equation}
+``` 
+where
+  - ``N_{rai}`` is the raindrops number density.
+Inserting this equation into the equation of ``lambda`` (equation 4 in Microphysics1M) yields
+```math
+\begin{equation}
+\lambda = \frac{1}{r_0}\left(\frac{ \Gamma(m_e + \Delta_m + 1) \, \chi_m \, m_0 \, N_{rai}}{q \, \rho}\right)^{\frac{1}{m_e + \Delta_m}}.
+\end{equation}
+```
+The values of terminal velocity and rain evaporation are then calculated by using the respective equations derived for the single moment scheme (equations 8 and 25 in Microphysics1M):
+```math
+\begin{equation}
+  v_t = \chi_v \, v_0 \, \left(\frac{1}{r_0 \, \lambda}\right)^{v_e + \Delta_v}
+        \frac{\Gamma(m_e + v_e + \Delta_m + \Delta_v + 1)}
+             {\Gamma(m_e + \Delta_m + 1)},
+\end{equation}
+```
+and
+```math
+\begin{align}
+\left. \frac{dq}{dt} \right|_{evap} =
+    \frac{4 \pi n_0}{\rho} (S - 1) G(T) \lambda^{-2}
+    \left(
+       a_{vent} +
+       b_{vent} \,
+         \left(\frac{\nu_{air}}{D_{vapor}} \right)^{\frac{1}{3}} \,
+         \left(\frac{1}{r_0 \, \lambda} \right)^{\frac{v_e + \Delta_v}{2}} \,
+         \left(\frac{2 \, \chi_v \, v_0}{\nu_{air} \, \lambda} \right)^{\frac{1}{2}} \,
+         \Gamma\left( \frac{v_e + \Delta_v + 5}{2} \right)
+    \right),
+\end{align}
+```
```

Comment:
Should there be a term for the rain drop number concentration sink due to evaporation?

Reply:
And for advection of rain drops with terminal velocity?

Reply:
We could consistently assume that the evaporation does not disappear any droplet, but only reduces the mass of each droplet. Or we can find the reduction of the number density of raindrops for drops larger than the assumed minimum raindrops mass x*.
Also for terminal velocity you're right, the advection velocity of number density should be number averaged. I'll find both the evaporation and the velocity tendencies for the rain number density and push more commits soon.

Reply:
Thank you, and apologies for not thinking about it sooner!